tslib/src/functions/check-validator/check-validator.ts

```diff
@@ -52,7 +43,7 @@ export class CheckValidatorV1 {
       this.detectArchive(receivedCheck)
       const decompressedString = this.compressionService.decompress(receivedCheck.archive)
       checkData = JSON.parse(decompressedString)
-      this.validateCheckStructure(checkData)
+      this.validateCheckStructureV2(checkData)
```

Comment:
It looks like this refactor could be finished by renaming the method back to the original? `validateCheckStructure`.  I'm not keen on versioned methods unless we absolutely need them.

Reply:
i'm not keen either.  i did a bit of refactoring to remove versioning from the whole function.  will fix